netutils/ftpc/ftpc_putfile.c

```diff
@@ -121,10 +172,11 @@ static int ftpc_sendbinary(FAR struct ftpc_session_s *session,
  ****************************************************************************/
 
 static int ftpc_sendtext(FAR struct ftpc_session_s *session,
-                         FAR FILE *linstream, FAR FILE *routstream)
```

Comment:
look like we don't need change the function prototype?

Reply:
I removed "routstream" argument from those three functions because it already presents inside of "session" argument structure. I think it was excessive (not reasonable).